docs/app/guides/getting-started/getting-started.component.html

```diff
@@ -0,0 +1 @@
+<div [markdown]="doc" [lineNumbers]="false"></div>
```

Comment:
Would be happy to chat about how best to format the markdown.  I've seen in the past that markdown docs usually have a "template" html file that the content is being inserted into, does the same apply here?  I didn't see a template but admittedly I didn't look all that hard.  One thing that will help the formatting out on this page is to wrap this div in `<div class="demo-content">` - either on here on in the template.  And then we should talk about how best to put this content into tiles as well.